### ISSUE TITLE
Bitcoin Price Prediction

### ISSUE THREAD

**Issue by prathimacode-hub** [DSC-JSS-NOIDA/ML-Hub#27]
• Objective : Create a model to predict the Bitcoin price
• Dataset : https://www.kaggle.com/datasets/vikramjeetsinghs/bitcoin-dataset
• Do Exploratory Data Analysis, create models using different algorithms and compare them using accuracy scores

**Comment by srivasPankhuri** [comment]
@prathimacode-hub if you wish to work on this dataset we can assign it to you.

**Comment by prathimacode-hub** [comment]
Yes, I would like to. Can you assign this issue to me? @srivasPankhuri

**Comment by srivasPankhuri** [comment]
Sure @prathimacode-hub

**Comment by Hrishikesh332** [comment]
Could you please assign this issue to me, I would like to work on this issue and also perform exploratory data analysis

**Comment by prathimacode-hub** [comment]
I'm working on it already. @Hrishikesh332
